hack/aws-image-job.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: aws-image-creation
+  namespace: openshift-sandboxed-containers-operator
+spec:
+  parallelism: 1
+  completions: 1
+  backoffLimit: 1
+  template:
+    metadata:
+      name: aws-image-creation
+# needs hostnet for ocp
+    spec:
+      volumes:
+      - name: shared-data
+        emptyDir: {}
+
+      hostNetwork: true # needed for imds access
+      securityContext:
+        runAsUser: 0 # needed for container mode dnf access
+
+      initContainers:
+      - name: payload
+        # change
+        image: quay.io/snir/podvm-binaries:rhel
+        volumeMounts:
+        - name: shared-data
+          mountPath: /payload
+        command: ["/bin/sh"]
+        args: ["-c", "cp /podvm-binaries.tar.gz /payload/ && cp /pause-bundle.tar.gz /payload/"]
+
+      containers:
+      - name: aws-image-creation
+        image: registry.access.redhat.com/ubi9/ubi:9.1
+        volumeMounts:
+        - name: shared-data
+          mountPath: /payload
+        env:
+          - name: PODVM_DISTRO
+            value: rhel
+          - name: INSTANCE_TYPE
+            value: "t2.small" # comment
```

Comment:
May be use the default value from the code and comment this variable.
Also mention about uncommenting and using a different instance if the default instance type is not available in the region.